bagit.py

```diff
@@ -751,8 +752,7 @@ def _make_manifest(manifest_file, data_dir, processes, algorithm='md5'):
             num_files += 1
             total_bytes += byte_count
             manifest.write("%s  %s\n" % (digest, _encode_filename(filename)))
-        manifest.close()
```

Comment:
The `with open(manifest_file, 'w') as manifest:` automatically closes the file when the block is done, even if there's an error, so `manifest.close()` is redundant.